tuxemon/event/actions/set_monster_level.py

```diff
@@ -29,36 +27,23 @@ class SetMonsterLevelAction(EventAction):
     """
 
     name = "set_monster_level"
-    level: int
-    slot: Union[int, None] = None
+    levels: int
+    slot: Optional[int]
 
     def start(self) -> None:
         if not self.session.player.monsters:
             return
 
-        def update_move(mon: Monster, level: int) -> None:
-            for move in mon.moveset:
-                if (
-                    move.technique not in (m.slug for m in mon.moves)
-                    and move.level_learned <= level
-                ):
-                    technique = Technique()
-                    technique.load(move.technique)
-                    mon.learn(technique)
-
-        monster_slot = self.slot
-        monster_level = self.level
-
-        if monster_slot:
-            if len(self.session.player.monsters) < int(monster_slot):
+        if self.slot is not None:
+            if len(self.session.player.monsters) < self.slot:
```

Comment:
Ok, usually, I'm kinda against comment, but I feel like we could describe what is happening there, or at least refactor it a bit.

In which case can the slot be bigger than the number of monsters the player owns?

Reply:
> In which case can the slot be bigger than the number of monsters the player owns?

This is simply if a modder put slot 5 when there are only 3 monsters.